### ISSUE TITLE
Timber Sentinel breaks token hud HP

### ISSUE THREAD

**Issue by RisingStarPF2e** [JDCalvert/pf2e-kineticists-companion#7]
Really weird bug with the Timber Sentinel feature I found in the 1.3.1 update

https://github.com/user-attachments/assets/581713b6-bdee-46ff-9ee5-e64031177bc5

- put timber sentinel on a scene
- now no matter what when you try to reduce HP via the Token Hud it refuses and spits an error about the tree.
- This breaks the token hud and you now cannot reduce HP via it, only on main sheets because it spits a error everytime.
- this is irregardless of where it is, what its status is, if the sheet exists or not, etc. 

Seems entirely related to the way it's referencing the actor/requiring it. **Tested with no modules on a fresh world results in the same.**

**TLDR: firing timber sentinel on any scene with the settings on bricks the token hud via it now checking every strike and spitting an error.**
- fix: Disable Timber Sentinel automation setting in module. (stops checking, returning function)

**Comment by RisingStarPF2e** [comment]
We SPEAKETH THY NAME! 🍡
